src/inspector_agent.cc

```diff
@@ -177,8 +175,36 @@ const std::string to_string(uint64_t number) {
   result << number;
   return result.str();
 }
-}  // namespace
 
+std::string StringViewToUtf8(const StringView& view) {
+  if (view.is8Bit()) {
+    return std::string(reinterpret_cast<const char*>(view.characters8()),
+                       view.length());
+  }
+  const uint16_t* source = view.characters16();
+  size_t result_length = view.length() * sizeof(*source);
+  std::string result(result_length, '\0');
+  UnicodeString utf16(reinterpret_cast<const UChar*>(source), view.length());
```

Comment:
If the reinterpret_cast is non-optional here, can you at least add a static_assert that checks the pointed-to types are of the same size?  See https://github.com/nodejs/node/pull/9280 for an example of what I mean.